src/foundations/type/Type.types.ts

```diff
@@ -0,0 +1,29 @@
+type TBodySize = 25 | 50 | 75 | 100 | 200 | 300;
+
+type TTypeSize = TBodySize | 400 | 500 | 600 | 700 | 800 | 900 | 1000;
+
+type TTypeHeadingElement = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+
+type TTypeBodyElement = 'p' | 'span';
```

Comment:
We might want to support a few other element here at some point, like `<label>` for instance. Maybe `<li>`? Don't worry about it right now, we can discuss later and update as needed.

Reply:
I added `label` for now. We need to extend them on a case-by-case basis.